core/store/src/trie/config.rs

```diff
@@ -8,8 +8,8 @@ use tracing::error;
 /// Default memory limit, if nothing else is configured.
 /// It is chosen to correspond roughly to the old limit, which was
 /// 50k entries * TRIE_LIMIT_CACHED_VALUE_SIZE.
-pub(crate) const DEFAULT_SHARD_CACHE_TOTAL_SIZE_LIMIT: u64 =
-    if cfg!(feature = "no_cache") { 1 } else { 50_000_000 };
+pub(crate) const DEFAULT_SHARD_CACHE_TOTAL_SIZE_LIMIT: bytesize::ByteSize =
+    if cfg!(feature = "no_cache") { bytesize::ByteSize::mb(1) } else { bytesize::ByteSize::mb(50) };
```

Comment:
Is the 1B -> 1MB intentional for the no cache case?

Reply:
ah, that was a mistake, thanks for catching!